api/WebSocket.json

```diff
@@ -224,7 +224,62 @@
       },
       "close": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/WebSocket/close",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/WebSocket/close_event",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": "12"
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": [
+              {
+                "version_added": "8"
+              },
+              {
+                "version_added": "4",
+                "version_removed": "8",
+                "notes": "Parameters not supported, see <a href='https://bugzil.la/674716'>bug 674716</a>."
+              }
+            ],
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            },
+            "webview_android": {
+              "version_added": null
```

Comment:
This should be true. Please double-check the webview support for the other socket events using this list: https://cs.chromium.org/chromium/src/android_webview/tools/system_webview_shell/test/data/webexposed/global-interface-listing-expected.txt?g=0&l=7724

Reply:
I think I've appropriately fixed this problem now.